Tak.py

```diff
@@ -1,21 +1,152 @@
 from collections import deque, namedtuple
+from enum import Enum
+from logging import raiseExceptions
 
 GameState = namedtuple('GameState', 'to_move, board, pieces, moves')
 
+class pieceType(Enum):
+        """"Enum enables easy access to standardized piece characteristics"""
+        BLACK, WHITE, WALL,TILE,CAPSTONE = range(5)
+
+class Piece:
+    """Tak game piece represents color, type, and XYZ position of piece"""
+    def __init__(self, color, type, position):
+        self.color = color
+        self.type = type
+        self.position = [position[0],position[1],position[2]] #row, col, dequeIndex
```

Comment:
What is standard Python convention for listing off parameters in terms of spaces? Are there typically no spaces after a comma? Right now we have spaces in some places in the file and none in others. It might help readability to be consistent throughout both the file and the project.

Reply:
Yes. Format consistency is important. My bad, will update with new commit.